src/main/java/uk/nhs/cdss/entities/SupplierPartitioned.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.cdss.entities;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+import lombok.Data;
+
+@Data
+@MappedSuperclass
+public class SupplierPartitioned {
+
+  @Column(name = "supplierToken")
+  private String supplierToken;
```

Comment:
This is probably more an ID than a token, since a token will be a signed supplier ID, and may change over time while retaining the same supplier ID.